src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -54,15 +54,25 @@ def trim_policy_path(policy):
 
     def get_organization_map(self, org_structure, counter=0):
         for name, ou_id in org_structure.copy().items():
+            if ou_id[0] not in ['r','o']:
+                continue
```

Comment:
I added a comment to this effect - I just wanted to avoid calling a `list_children` operation on an account (which doesn't have children); but perhaps this is an unnecessary check, and the `list_children` operations would just return an empty lists instead of erroring in some way.